src/com/twitter/intellij/pants/execution/PantsMakeBeforeRun.java

```diff
@@ -163,6 +163,15 @@ public boolean canExecuteTask(
     return true;
   }
 
+  public static class PantsExecuteTaskResult {
```

Comment:
How about moving this into a separate file as a standalone class? because it is a bit awkward for a lot of the other class to depend on an inner class e.g.
```
protected PantsMakeBeforeRun.PantsExecuteTaskResult pantsCompileProject() {
...
}
```

Reply:
moved into a standalone class